src/tests/issues.targets

```diff
@@ -1184,14 +1184,8 @@
         <ExcludeList Include="$(XunitTestBinBase)/Loader/classloader/StaticVirtualMethods/InterfaceVariance/**">
             <Issue>https://github.com/dotnet/runtime/issues/88689</Issue>
         </ExcludeList>
-        <ExcludeList Include="$(XunitTestBinBase)/Regressions/coreclr/GitHub_85240/test85240/**">
-            <Issue>https://github.com/dotnet/runtime/issues/71095</Issue>
-        </ExcludeList>
-        <ExcludeList Include="$(XunitTestBinBase)/JIT/opt/Remainder/Regressions/Regression1/**">
-            <Issue>https://github.com/dotnet/runtime/issues/79022</Issue>
-        </ExcludeList>
```

Comment:
This is correct. `JIT/opt/Remainder/Regressions/Regression1` is now passing with Mono.
Thank you for cleaning this up.